source/_integrations/google_translate.markdown

```diff
@@ -16,8 +16,81 @@ The `google_translate` text-to-speech platform uses the unofficial [Google Trans
 
 {% include integrations/config_flow.md %}
 
-Check the [complete list of supported languages](https://cloud.google.com/translate/docs/languages) (languages where "Talk" feature is enabled in Google Translate) for allowed values.
-Use the 2-digit language code which you can find at the end of the URL when you click on the language name.
+<details>
+<summary><b>Supported Languages </b></summary>
+  
+All languages where "Talk" feature is enabled in Google Translate are supported. The following is the current list of languages supported by Google. 
+If you'd like to add a newly supported language that is not in the list, follow the directions outlined in [this file](https://github.com/home-assistant/core/blob/dev/homeassistant/components/google_translate/const.py).
```

Comment:
```suggestion
```
This documentation is aimed at end users. I don't think this is something we want end users to do.